apps/typegpu-docs/src/examples/algorithms/concurrent-chart/index.ts

```diff
@@ -0,0 +1,145 @@
+import tgpu from 'typegpu';
+import { performCalculationsWithTime } from './calculator.ts';
+
+const bars = Array.from(document.querySelectorAll<HTMLDivElement>('.bar'));
```

Comment:
I think this file could be cleaned up a little, for example this could help

```suggestion
const barElements = document.querySelectorAll<HTMLDivElement>('.bar');
const bars = Array.from({ length: 5 }, (_, i) => ({
  jsBar: barElements[i * 3 + 0],
  gpuTotalBar: barElements[i * 3 + 1],
  gpuBar: barElements[i * 3 + 2],
}));
```